<section class="slice slice-lg sct-color-1">
  <div class="container">
    <div class="row">
      <div class="col-md-3"></div>
      <div class="col-md-6">
        <div class="form-user style-1 rounded has-shadow-1">
          <div class="login-signup-title">新規投稿フォーム</div>
          <div class="form-body">
            <%= form_with model:@post, url: posts_path, local: true do |f| %>
              <% if @post.errors.any? %>
                <div id="error_explanation">
                  <h2><%= @post.errors.count %>件のエラーが発生しました。</h2>
                  <ul>
                  <% @post.errors.full_messages.each do |message| %>
                    <li><%= message %></li>
                  <% end %>
                  </ul>
                </div>
              <% end %>
              <div class="row">
                <div class="col-sm-2"></div>
                <div class="col-sm-8">
                  <%= attachment_image_tag @post, :image, class: "img-square", format: 'jpeg', fallback: "no_image.jpg", width:"80%" %>
                  <%= f.attachment_field :image, placeholder: "画像", class: "image-prev" %>
                </div>
                <div class="col-sm-2"></div>
              </div>
              <br>
              <div class="row">
                <div class="col-sm-12">
                  <%= f.text_area :caption, class:"form-control", rows:"3", placeholder:"コメントここに", id: :caption %>
                </div>
              </div>
              <hr>
              <%= f.submit "送信", class:"btn btn-info btn-block" %>
            <% end %>
          </div>
        </div>
      </div>
      <div class="col-md-3"></div>
    </div>
  </div>
</section>